src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.csproj

```diff
@@ -13,15 +13,15 @@
     <PackageReference Include="Dapper" Version="2.1.66" />
     <PackageReference Include="DuckDB.NET.Data.Full" Version="1.3.2" />
     <PackageReference Include="InterpolatedSql.Dapper" Version="2.4.0" />
-    <PackageReference Include="linq2db.EntityFrameworkCore" Version="8.1.0" />
+    <PackageReference Include="linq2db.EntityFrameworkCore" Version="8.0.0" />
```

Comment:
This looks like a downgrade of `linq2db.EntityFrameworkCore` from the current version which is `8.1.0`.

What's the reason for that?